Apps/WebClient/src/ClientApp/src/components/timeline/linearTimeline.vue

```diff
@@ -264,5 +268,18 @@ export default class LinearTimelineComponent extends Vue {
         );
         this.currentPage = Math.floor(index / this.numberOfEntriesPerPage) + 1;
     }
+
+    private addLogging(): void {
+        const logger: ILogger = container.get(SERVICE_IDENTIFIER.Logger);
```

Comment:
this is an interesting example. We can check this out later if we want a full trace of the methods. For now I think the basic logger.debug where necessary should suffice

Reply:
exactly, we don't have to go all-in into this for now. The question may be if we want to merge this code to dev or just keep it as an example on this branch only.